bVNC/src/main/java/com/iiordanov/bVNC/dialogs/RateOrShareFragment.kt

```diff
@@ -60,10 +62,53 @@ class RateOrShareFragment : DialogFragment() {
         }
         versionAndCode = v.findViewById<View>(R.id.versionAndCode) as TextView
         versionAndCode?.text = Utils.getVersionAndCode(v.context)
-
+        setVisibilityOfElements(v)
         return v
     }
 
+    fun setTitle() {
+        if (Utils.isCustom(context)){
+            try {
+                dialog?.setTitle(
+                    getString(
+                        requireContext().resources.getIdentifier(
+                            Utils.getStringConfigAttribute(App.configFileReader.configData, TAG.replaceFirstChar { it.lowercase() }, "title", "key"),
+                            "string",
+                            Utils.pName(context)
+                        )
+                    )
+                )
+                return
```

Comment:
I prefer having no random returns within methods. In both cases you're calling dialog?.setTitle() with a string parameter. Have the if/else statement set that string parameter appropriately, and call dialog?.setTitle(thatStringParameter) at the end of the method.